src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NETFX_CORE || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS_UWP
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS || WINDOWS_WINUI || WINUI
```

Comment:
Are there platforms where the behavior is not supported? Couldn't we just remove this `#if`?

Reply:
It looks like it's supported on all platforms, I've removed the `#if`